src/modules/Servicedomain/html_client/mod_servicedomain_manage.html.twig

```diff
@@ -1,325 +1,322 @@
+                        </div>
 
-                            </fieldset>
-                        </form>
+                        <div class="col-md-5">
+                            <label for="whois_postcode" class="form-label">{{ 'Zip'|trans }}</label>
+                            <input class="form-control" type="text" name="contact[postcode]" id="whois_postcode"
+                                   value="{{ request.postcode|default(service.contact.postcode) }}" required>
+                        </div>
                     </div>
-                </div>
-                <div class="tab-pane" id="epp">
-                    <div class="block">
-                        <h2>{{ 'Domain Secret'|trans }}</h2>
-                        <p class="alert alert-info">{{ 'All domain names (except a .EU, .UK domain name) have a Domain (Transfer) Secret Key/Authorization Code (EPP Code) associated with them. This would be required if you want to transfer service.'|trans }}</p>
-                        <div class="control-group">
-                            <div class="controls">
-                                <button class="btn btn-primary" type="button" id="get-epp">{{ 'Get EPP code'|trans }}</button>
-                            </div>
+                    <div class="mb-3 row">
+                        <div class="col-md-5">
+                            <label for="whois_phone_cc" class="form-label">{{ 'Phone Country Code'|trans }}</label>
+                            <input class="form-control" type="text" name="contact[phone_cc]" id="whois_phone_cc"
+                                   value="{{ request.phone_cc|default(service.contact.phone_cc) }}" required>
+                        </div>
+                        <div class="col-md-5">
+                            <label for="whois_phone" class="form-label">{{ 'Phone number'|trans }}</label>
+                            <input class="form-control" type="text" name="contact[phone]" id="whois_phone"
+                                   value="{{ request.phone|default(service.contact.phone) }}" required>
                         </div>
+                    </div>
+                    <input type="hidden" name="order_id" value="{{ order.id }}">
+                    <button class="btn btn-primary" type="submit" value="{{ 'Update' }}">{{ 'Update'|trans }}</button>
+                </form>
+            </div>
+
+            {# Domain transfer #}
+            <div class="card-body tab-pane" id="transfer-tab-pane" role="tabpanel" tabindex="0">
+                <div class="mb-3">
+                    <h5 class="mt-1 mb-1">{{ 'Domain Secret'|trans }}</h5>
+                    <span class="small text-secondary">{{ 'All domain names (except a .EU, .UK domain name) have a Domain (Transfer) Secret Key/Authorization Code (EPP Code) associated with them. This would be required if you want to transfer service.'|trans }}</span>
```

Comment:
I suggest to change the text to: 
{{ 'All domain names (except some special cases like .EU and .UK domains) have a Domain (Transfer) Secret Key/Authorization Code (Auth-Code) associated with them. This code is required if you want to transfer your domain to another service.'|trans }}

Also, maybe we should add something like this: 

<div class="m-2 alert alert-info" id="information-block"><strong>{{ 'Important!'|trans }}</strong>{{'Keep this code secure, and don not share it with others'|trans}}</div>